pkg/api/machine.go

```diff
@@ -28,13 +28,16 @@ type MachineSpec struct {
 	NetworkInterfaces []*NetworkInterfaceSpec `json:"networkInterfaces"`
 
 	ShutdownAt time.Time `json:"shutdownAt,omitempty"`
+
+	QemuGuestAgentEnable bool
```

Comment:
```suggestion
	QemuGuestAgent GuestAgent
```

Can we maybe update the spec? 
```
type GuestAgent string 
const (
	EnableGuestAgent GuestAgent = "Enable"
	DisableGuestAgent GuestAgent = "Disable"
)
```

Reply:
What are benefits? Because we have some approach for Hugepages. Do will we need more values as true/false? Compare of strings can has higher impact to performance too.

Reply:
These are the k8s api conventions. The general reasoning is that the string types can be extended if needed.

Reply:
we can go this way, but i prefer lower case letters. It is simpler for comparison and i don't have to do conversation or support multiple variant.

Reply:
i  will implement it as you suggest for keep the same format. Conversion to lower case we can discuss next time.

Reply:
fixed